packages/react-ui-components/src/SelectBox_Option_SingleLine/selectBox_Option_SingleLine.js

```diff
@@ -13,21 +13,27 @@ class SelectBox_Option_SingleLine extends PureComponent {
         }).isRequired,
 
         disabled: PropTypes.bool,
+        showIcon: PropTypes.bool,
 
         className: PropTypes.string
     }
 
     render() {
-        const {option, className, disabled} = this.props;
+        const {option, className, disabled, showIcon, icon} = this.props;
 
         const isDisabled = disabled || option.disabled;
 
         const finalClassNames = mergeClassNames({
             [className]: className
         });
 
+        // We want to show the icon of the option when it is defined
+        // If we have no option icon but showIcon is true, we want to show the icon of the SelectBox
+        // If we have no option icon and showIcon is false, we want to show no icon
+        const visibleIcon = option.icon ? option.icon : (showIcon ? icon : null);
```

Comment:
What about icon is set but showIcon is false?
Sounds confusing?

Reply:
Adjusted this part as well.